selfdrive/car/tesla/readconfig.py

```diff
@@ -183,7 +201,7 @@ def read_config_file(CS):
       file_changed = True
     config.set('OP_CONFIG', 'use_analog_when_no_eon', CS.useAnalogWhenNoEon)
     
-
+    #do_auto_update - Set this setting to False if you do not want OP to autoupdate every time you reboot and there is a change on the repo

```

Comment:
I would also like to disable this one, but maybe it's good for people? haha. I prefer doing it manually, but I can see how that's not very user friendly. so I left it enabled